src/main/java/de/unistuttgart/iste/gits/content_service/persistence/repository/ContentRepository.java

```diff
@@ -0,0 +1,28 @@
+package de.unistuttgart.iste.gits.content_service.persistence.repository;
+
+import de.unistuttgart.iste.gits.content_service.persistence.dao.ContentEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface ContentRepository extends JpaRepository<ContentEntity, UUID> {
+
+    List<ContentEntity> findByName(String name);
+
+    @Query("select t.content from Tag t where t.name = :tag")
```

Comment:
For me, it is less a question about sufficiency but more about readability. I often prefer manually defined queries because it is very clear what happens. Do you think a comment would really help in this case?

Reply:
I see your point. I'm just a bit careful with SQL. If something goes wrong, it's another thing that might be debugged.